tests/ttnn/unit_tests/operations/test_moreh_softmax.py

```diff
@@ -385,6 +381,13 @@ def test_softmax_backward_callback(shape_dim_strategy, device):
     y.backward(dy)
     for i in range(2):
         tt_npu = ttnn.operations.moreh.softmax_backward(dev_y, dev_dy, dim, strategy=strategy)
+        if i == 0:
+            num_program_cache_entries = device.num_program_cache_entries()
+            assert num_program_cache_entries > 0
+        else:
+            assert device.num_program_cache_entries() == num_program_cache_entries
+        torch_dummy = torch.randn([32, 32])
+        tt_dummy = to_npu(torch_dummy, device)
```

Comment:
It seems that `softmax backward` has the same issue as `softmin backward`.

Reply:
Resolved

Reply:
```Python
 for i in range(2):
        x = torch.randint(low=0, high=4, size=shape).to(torch.bfloat16)

        dev_x = ttnn.Tensor(x, ttnn.bfloat16).to(ttnn.TILE_LAYOUT).to(device)

        tt_cpu = torch.softmax(x, dim)
        dev_x = ttnn.Tensor(x, ttnn.bfloat16).to(ttnn.TILE_LAYOUT).to(device)
```
It seems that the code for creating dev_x is unnecessarily duplicated twice.